data/layouts/huion-new-1060-plus.svg

```diff
@@ -0,0 +1,180 @@
+<?xml version="1.0" standalone="no"?>
+<!DOCTYPE svg PUBLIC "-//W3C//DTD SVG 1.1//EN"
+   "http://www.w3.org/Graphics/SVG/1.1/DTD/svg11.dtd">
+<svg
+   width="360.0"
+   height="240.0"
+   style="color:#000000;font-size:8px;fill:none;stroke:#7f7f7f;stroke-width:0.25"
+   version="1.1"
+   xmlns="http://www.w3.org/2000/svg">
+  <title
+     id="title">Huion New 1060 Plus</title>
+  <path
+     id="ButtonA"
+     class="A Button"
+     d="M 16.0 42.5 H 32.0 v 12.0 H 16.0 c -0.5 0.0 -1.0 -0.5 -1.0 -1.0 v -10.0 c 0.0 -0.5 0.5 -1.0 1.0 -1.0 z" />
+  <path
+     id="LeaderA"
+     class="A Leader"
+     d="M 8.5 51.5 H 13.5" />
+  <text
+     id="LabelA"
+     class="A Label"
```

Comment:
The README says that I should add `ModeSwitch` if this is a ModeSwitch button. But I have no idea what this is.

Reply:
The mode switch button changes the mode of the ring/strips (if any) and toggles the LEDs arround too. So you can have the ring scroll up/down in mode 1 but zoom in/out in mode 2. Doesn't apply here, so it's not a mode switch.